### PR TITLE
Add skiplink story

### PR DESCRIPTION
**Pull request by hariscs**
## Describe your changes
created skiplink story

## Screenshots - If Any (Optional)
image

## Link to issue

Add skiplink story #164 

## Checklist before requesting a review

- [x] I have performed a self-review of my code.
- [x] Followed the repository's [Contributing Guidelines](/CONTRIBUTING.md).
- [x] I ran the app and tested it locally to verify that it works as expected.
- [x] I have checked my code with an automatic accessibility tool such as Axe Dev Tools or Wave 
      and it shows no errors.
## Additional Information (Optional)
Any additional information that you want to give us.

<a href="https://gitpod.io/#https://github.com/AccessibleForAll/AccessibleWebDev/pull/180"></a>

### CODE REVIEW

File: stories/SkipLink.stories.tsx [hunk 1]
```diff
@@ -0,0 +1,12 @@
+import React from "react"
+import { ComponentStory, ComponentMeta } from "@storybook/react"
+import { SkipLink } from "../components/SkipLink/SkipLink"
+
+export default {
+	title: "Example/Header",
```
Comment (ctoffanin) [inline stories/SkipLink.stories.tsx:6]: The name of the story should be "Example/SkipLink"

**Review by ctoffanin — CHANGES_REQUESTED** [review]

**Comment by ctoffanin** [comment]
Thank you for contributing!
There is only a small detail to fix, and we can merge it.

File: stories/SkipLink.stories.tsx (hunk 1 above)
Comment (hariscs) [reply stories/SkipLink.stories.tsx]: done

**Review by ctoffanin — APPROVED** [review]

**Comment by ctoffanin** [comment]
Nice, perfect!

**Pull request merged**